<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1270</width>
    <height>823</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1261</width>
      <height>821</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="4,1">
     <property name="spacing">
      <number>6</number>
     </property>
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>1</number>
       </property>
       <widget class="QWidget" name="tab_5">
        <attribute name="title">
         <string>Tab 1</string>
        </attribute>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>49</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_3">
        <attribute name="title">
         <string>Page</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="tab_6">
        <attribute name="title">
         <string>Tab 2</string>
        </attribute>
        <widget class="QLabel" name="label_5">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>0</y>
           <width>49</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QTextBrowser" name="statusText"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Resolotion</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="reso">
           <property name="text">
            <string>1000</string>
           </property>
           <property name="placeholderText">
            <string>1000</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>FPS</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="fps">
           <property name="value">
            <number>2</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLineEdit" name="fileName"/>
         </item>
         <item>
          <widget class="QPushButton" name="fileChoose">
           <property name="text">
            <string>Choose</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>frame</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="frames"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="startBtn">
         <property name="text">
          <string>START</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="stopBtn">
         <property name="text">
          <string>STOP</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="restartBtn">
         <property name="text">
          <string>RESTART</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>Generate QR Code</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
